README.md

```diff
@@ -1 +1,77 @@
-# README
+# Plan-app
+
+## Description
+
+This Ruby on Rails project is a mobile web application where you can manage your budget, you have a list of transactions associated with a category so that you can see how much money you spent and on what. It allows the user to: register and log in, so that the data is private to them, introduce new transactions associated with a category, and see the money spent on each category.
+
+## Demo
+
+### Video Presentation
+
+[Live Demo](https://salty-stream-24790.herokuapp.com/)
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the updated version of the app gets properly deployed to Heroku so that anyone how visits the URL can easy navigate through the app. If the issue you have persists, kindly take down this deployment and try redeploying. the currently deployed version does not allow users to add any category.
![Screenshot from 2022-05-28 00-41-00](https://user-images.githubusercontent.com/58692590/170801767-da4ed07f-a1bb-4e77-90d8-29041dd5438f.png)